common/util_test.go

```diff
@@ -1043,3 +1044,64 @@ func TestDeserializeSearchAttributeValue_Error(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateLongPollContextTimeout(t *testing.T) {
```

Comment:
This could be turned into table formatted test to reduce repeated test body code. 
```
tests := []struct {
  desc string
  loggerFn func() log.MockLogger
  timeout time.Duration
  wantErr error
} {
...
}
```

Reply:
Yeah, it can be turned, but I am not sure the readability will be improved. Also, 2 of 4 cases check errors, so it requires us to write conditionals. I'd like to leave it as it is.